test_projects/rails/rspec_rails/Gemfile

```diff
@@ -14,36 +14,46 @@ else
 end
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem "rails", "~> 6.0.2", ">= 6.0.2.1"
+gem 'rails', '~> 6.1.0'
 # Use Puma as the app server
-gem "puma", "~> 4.1"
+gem 'puma', '~> 5.0'
```

Comment:
We use `"` in the main Gemfile so I'd prefer it stays the same here and not changed here :)